hr_timesheet_sheet/views/account_analytic_line_views.xml

```diff
@@ -102,7 +102,7 @@
     <menuitem
         action="act_hr_timesheet_line_to_submit_my"
         id="menu_act_hr_timesheet_line_to_submit_my"
-        parent="hr_timesheet.menu_hr_time_tracking"
+        parent="hr_timesheet_sheet.menu_hr_my_timesheets"
```

Comment:
I think it should be `hr_timesheet.timesheet_menu_activity_user` because https://github.com/odoo/odoo/blob/15.0/addons/hr_timesheet/views/hr_timesheet_views.xml#L313-L317

Reply:
The menu item `hr_timesheet.timesheet_menu_activity_user` is now inside the `hr_timesheet_sheet.menu_hr_my_timesheets` menu, and it is also related to an action, so maybe we shouldn't be using it as a parent in this menu item. Why do you think it should be like that?

Reply:
Yeah, I concur